chapters/VK_EXT_acquire_drm_display/acquire_drm_display.txt

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018-2021 The Khronos Group, Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[open,refpage='vkAcquireDrmDisplayEXT',desc='Acquire access to a VkDisplayKHR using DRM',type='protos']
+--
+
+To acquire permission to directly a display in Vulkan from the Direct Rendering
+Manager (DRM) interface, call:
+
+include::{generated}/api/protos/vkAcquireDrmDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device the display is on.
+  * pname:drmFd Connection to the DRM node file descriptor.
+  * pname:display The display the caller wishes Vulkan to control.
+
+All permissions necessary to control the display are granted to the Vulkan
+instance associated with the provided pname:physicalDevice until the display is
+either released or the connector is unplugged.
+If any error is encountered during the acquisition of the display, the call
+must return the error code ename:VK_ERROR_INITIALIZATION_FAILED.
+
+The provided DRM fd should not be closed before the display is released,
+attempting to do it may result in undefined: behaviour.
+
+include::{generated}/validity/protos/vkAcquireDrmDisplayEXT.txt[]
+--
+
+[open,refpage='vkGetDrmDisplayEXT',desc='Query the VkDisplayKHR corresponding to a DRM connector ID',type='protos']
+--
+
+Before acquiring a display from the DRM interface, the caller may want to
+select a specific sname:VkDisplayKHR handle by identifying it using a
+pname:connectorId. To do so, call:
+
+include::{generated}/api/protos/vkGetDrmDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device to query the display from.
+  * pname:drmFd Connection to the DRM node file descriptor.
```

Comment:
Ditto, remove "connection" here.

DRM master is not strictly needed here, since this is just used for information gathering purposes.